package.json

```diff
@@ -11,7 +11,23 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "stylelint": "stylelint **/*.css",
-    "stylelint:fix": "stylelint --fix **/*.css"
+    "stylelint:fix": "stylelint --fix **/*.css",
+    "prepare": "husky install"
+  },
+  "husky": {
```

Comment:
I think that this is not required but please double check it official documentation.

Reply:
You are right, there is no need to specify this.